README.md

```diff
@@ -29,3 +29,8 @@ print $ulid; // 01b8kyr6g8bc61ce8r6k2t16hy
 ## License
 
 MIT © [Robin van der Vleuten](https://www.robinvdvleuten.nl)
+
+
```

Comment:
Extraneous newline + could you move the section above the license so that stays at the bottom?

Reply:
Sure! :-)